txscript/standard_test.go

```diff
@@ -882,7 +913,7 @@ var scriptClassTests = []scriptClassTest{
 	{
 		// Nulldata with more than max allowed data (so therefore
```

Comment:
Seems to have lost the updates from the upstream commit to this comment.

Reply:
maintained the script for that test because the one from upstream seemed to be was categorized as `nulldata`. I suspect it's because the script doesn't exceed `MaxDataCarrierSize`.

```--- FAIL: TestScriptClass (0.00s)
	standard_test.go:996: nulldata exceed max standard push: expected nonstandard got nulldata (script 6a4c51046708afdb0fe5548271967f1a67130b7105cd6a828e03909a67962e0ea1f61deb649f6bc3f4cef3046708afdb0fe5548271967f1a67130b7105cd6a828e03909a67962e0ea1f61deb649f6bc3f4cef308)
```

Reply:
Right, but the comment update itself still applies.

Reply:
oh right, updating.